components/Sidebar/ClientSidebar.tsx

```
@@ -535,11 +578,17 @@ function ClientSidebarContent({
   // Now let's add the optimistic UI update functionality
   // This function will be used to add a new chat to the list immediately when user creates one
   const addOptimisticChat = useCallback(
-    (chatTitle: string, moduleId: string | null = null) => {
+    (
+      chatTitle: string,
+      moduleId: string | null = null,
+      forceOldest: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)